### DISCUSSION TITLE
Home Assistant Yellow mit CM4 und HM-MOD-RPI-PCB

### DISCUSSION THREAD

**Discussion by Sechskant-Modelle** [OpenCCU/OpenCCU#2251] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe eine Home Assistant Yellow mit einem CM4 im Einsatz. Zum Betrieb der RapsberryMatic möchte ich ein HM-MOD-RPI-PCB nutzen. Mein Problem ist aktuell das das Modul nicht innerhalb der RaspberryMatic im Posteingang angezeigt wird damit ich es einrichten könnten.

Beim booten wir das Modul korrekt erkannt

> Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: onboard, OK
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.0, OK
Identifying Homematic RF-Hardware: ....HmRF: HM-MOD-RPI-PCB/GPIO@fe201000.serial, HmIP: HM-MOD-RPI-PCB/GPIO@fe201000.serial, OK
Updating Homematic RF-Hardware: HM-MOD-RPI-PCB: 2.8.6, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: creating new SSL cert... OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .......OK
Starting rfd: .OK
Starting HMIPServer: .............OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: OK
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.67.10.20230225 (raspmatic_oci_arm64)``

Auch in der INFO wird das korrekt angezeigt

> CCU Hardware Info
Serial Number:	SEQ1773606
Hardware Model:	n/a (oci)
CPU / Memory:	Cortex-A72 (4), 7.7G
Storage:	931.5G WD Blue SN570 1TB /dev/nvme0n1
Real-Time-Clock:	onboard
CCU Software Info
Product:	raspmatic_oci_arm64 (3.67.10.20230225)
ReGaHss:	R1.00.0388.0233
Engines:	Tcl (8.6.12), Java (1.8.0_362), NodeJS (v16.18.1)
Status:	SD(1) NTP(0) Link(1) Internet(1) IP(1)
Operating System Info
OS Type/Kernel:	Buildroot 2022.11.1 (Linux 5.15.84-v8 aarch64)
Uptime:	0 d, 0 h 32 min
Load Average:	0.28 0.19 0.25
System Temperature:	38.5 °C
Memory / Swap Utilization:	15.4 % / 0.0 %
NTP Offset (Server):	n/a (n/a)
rootfs / userfs Free Space:	875.6 GB [95.5%] / 875.6 GB [95.5%]
homematicIP-RF (HmIP) Info
RF-Module/Firmware:	HM-MOD-RPI-PCB (2.8.6)
Device-Node:	/dev/raw-uart (GPIO@fe201000.serial)
Address:	0xBCA93D (0xBCA93D)
SGTIN:	3014F711A061A7DD89A47826
Serial:	SEQ1773606
DC / CS:	1 % / -1 %
HomeMatic-RF (HmRF) Info
RF-Module/Firmware:	HM-MOD-RPI-PCB (2.8.6)
Device-Node:	/dev/raw-uart (GPIO@fe201000.serial)
Address:	0x752154 (0x752154)
Serial:	SEQ1773606
DC / CS:	1 % / -1 %

Die Settings inder config.txt habe ich auch wie folgt gemacht

> enable_uart=1
dtparam=i2c_arm=on
dtoverlay=miniuart-bt
dtoverlay=rpi-rf-mod

Auf der Yellow Platine nutze ich den 3.3V Pin sowie GND und RX/TX 

Allerdings erscheint das Gerät nicht im Posteingang womit ich es auch nicht final einrichten kann. Gibt es noch Idee woran es liegen könnte?

**Comment by Baxxy13** [comment, marked as answer]
Ein HM-MOD-RPI-PCB landet niemals im Posteingang und muss auch nicht eingerichtet werden.
Es erscheint auch nicht als Gerät in der WebUI.
Gucke auf die Hilfeseite innerhalb der WebUI oder gucke im Anlern-Fenster ob beide Anlernmodi (HM /HmIP) verfügbar sind.

**Reply by Sechskant-Modelle** [reply]
Danke! Manchmal sieht man den Wald vor lauter Bäumen nicht - beide Anlernmodi sind verfügbar

**Comment by bogosavljevic** [comment]
Hi All,

Is it possible to share some pictures how you connected HM-MOD-RPI PCB with board.

Many thanks,
Dimitrije

**Reply by Baxxy13** [reply]
Maybe this helps...
![HA_Yellow_HM-MOD-RPI-PCB](https://github.com/jens-maus/RaspberryMatic/assets/54898249/894d51e7-de5b-4ea8-b237-4f5ee44912ae)
